### ISSUE TITLE
Online translator www.deepl.com does not work in Mypal 68.12.3

### ISSUE THREAD

**Issue by AstroSkipper** [Feodor2/Mypal68#75]
The Online translator **https://www.deepl.com/en/translator** does not work at all. Some elements of this site like language settings and buttons are not shown and input text boxes don't work.

**Comment by RamonUnch** [comment]
For me it works fine...

**Comment by AstroSkipper** [comment]
> For me it works fine...

You're right. Today it works for me again. Maybe a temporary effect. Issue will be closed.

**Comment by AstroSkipper** [comment]
Temporary problem. Issue resolved.
